conf/empire/empire.yaml

```diff
@@ -186,3 +186,19 @@ stacks:
     parameters:
       ServiceToken: empireDaemon::CustomResourcesTopic
       Name: "acme-inc"
+  - name: acmeAppEnvironment
+    class_path: stacker_blueprints.empire.app.AppEnvironment
+    parameters:
+      ServiceToken: empireDaemon::CustomResourcesTopic
+      AppId: acmeApp::AppId
+      Variables:
+        FOO: FOO
+        APP_ID: acmeApp::AppId
+        DATABASE_URL:
```

Comment:
Maybe we could add support for something like:

`DATABASE_URL: postgres://${empiredb_user}:${empiredb_password}@${{empireDB::DBCname}}/empire`

where `${...}` is the default Template string replacement that will get resolved when we parse the config on start, but `${{...}}` is a value that will get resolved during build time.